crates/bevy_text/src/font_atlas_set.rs

```diff
@@ -67,10 +75,30 @@ impl FontAtlasSet {
             if let Some(last_glyph) = last_glyph.take() {
                 width += scaled_font.kern(last_glyph.id, glyph.id);
             }
-            if font_atlas.get_char_index(character).is_none() {
+            if !font_atlas
+                .iter()
+                .any(|atlas| atlas.get_char_index(character).is_some())
+            {
                 if let Some(outlined_glyph) = scaled_font.outline_glyph(glyph.clone()) {
                     let glyph_texture = Font::get_outlined_glyph_texture(outlined_glyph);
-                    font_atlas.add_char(textures, texture_atlases, character, &glyph_texture);
+                    let add_char_to_fontatlas = |atlas: &mut FontAtlas| -> bool {
```

Comment:
nit: as long as it is, naming conventions dictate that this should be `add_char_to_font_atlas` :)

Reply:
agreed - although I hate putting it into a Fn ... but clippy complained :-(

Reply:
bawhaha yeah sometimes clippy is overly opinionated imo.